docs/iceberg-rest-service.md

```diff
@@ -1,470 +0,0 @@
----
```

Comment:
Was this document deleted by mistake?

Reply:
Thanks for the review.
This file was renamed into `admin\iceberg-server.md`.